docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
+# Deploy Hyperlane to a new chain
 
-# Get Started
-
-Hyperlane can be deployed by anyone to any chain.
-This guide covers sending your first interchain message from a new EVM chain. If you'd like to deploy to a Cosmos appchain, follow this [community-submitted guide](https://github.com/many-things/cw-hyperlane/blob/25a31818e9bf909875ab091f3884ee71bd25814f/DEPLOYMENT.md).
+:::info
+This guide will help you deploy Hyperlane to your new chain as quickly as possible. This includes the core mailbox and ISM contracts as well as warp route contracts for assets you’re bridging
 
-By the end of this guide you will have deployed and configured your Mailbox smart contract and off-chain agents, allowing developers to send interchain messages to and from your chain.
+The guide is intended for testing - by the end you’ll be able to pass messages between your custom chain and any other in the Hyperlane network.
 
-:::note
-
-This guide has been specifically made to demonstrate how to connect a chain that _already has Hyperlane on it_ to a new chain that does _not_ have Hyperlane.
+To go to production, ensure you’ve successfully pushed a registry PR (the last step) and reach back over telegram and we can add your custom chain to the core chains we have in the CLI and relay for.
 
 :::
 
-<TerminologyPartial />
+## Pre-Requisites: Chain Metadata & Funded Wallet
 
-## Overview
+Anyone can begin this quickstart guide once they have the following artifacts & assets available:
+- A new, custom, or pre-existing network of your choice, including the following metadata:
+    - A chain name, e.g. `ethereum`
+    - A chain ID, e.g. `1`
+    - A RPC URL, e.g. `https://eth.llamarpc.com`
+- A deployer wallet/EOA private key or seed phrase
+    - This EOA should be funded on your custom chain and any chain you will be passing messages to & from
+- The Hyperlane CLI npm package installed (see below)
 
-There are five steps in this guide:
+## 1) Install the Hyperlane CLI
 
-1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayer.
-2. [<b>Deploy contracts</b>](#2-deploy-contracts) to the local chain and to every remote chain with which the local chain will be able to send and receive messages.
-3. [<b>Run validators and relayer </b>](#3-run-validators-relayer) using [Kurtosis](https://cloud.kurtosis.com). Validators provide the signatures needed for the Interchain Security Modules you deployed in step 2. The relayer will deliver messages between chains you deployed contracts to.
-4. [<b>Send a test message</b>](#4-send-test-messages) to confirm that your relayer is able to deliver messages to and from each pair of chains
-5. [<b>Deploy a warp route</b>](#5-optional-deploy-a-warp-route) to send token value, not just messages, across chains
+To install the Hyperlane CLI npm package globally, run:
 
-## Getting Started
+```bash
+npm i -g @hyperlane-xyz/cli
```

Comment:
They'll need node18 or newer to run this, consider including a tip for that. Or better yet, avoid duplicating these instructions and instead just link to the CLI page like the Deploy Warp Route page does